cerberus-web/src/main/java/com/nike/cerberus/service/AuthenticationService.java

```diff
@@ -221,7 +221,10 @@ public EncryptedAuthDataWrapper authenticate(IamRoleCredentials credentials) {
 
     final String iamPrincipalArn =
         String.format(
-            AWS_IAM_ROLE_ARN_TEMPLATE, credentials.getAccountId(), credentials.getRoleName());
+            AWS_IAM_ROLE_ARN_TEMPLATE,
+            "aws",
```

Comment:
There is a chance. Right now it's only there for backwards compatibility. Maybe we can use aws/aws-cn depending on the feature flag or adding an extra field in the V1 KMS controller. But since the KMS is deprecated, I'd avoid putting too much working into it.